configure.cmd

```diff
@@ -111,12 +120,15 @@ REM Parse command line parameters
             set USE_CLANG_CL=1
         ) else if /I "%1" == "VSSolution" (
             set VS_SOLUTION=1
-            REM explicitly set VS version for project generator
+            
+			REM explicitly set VS version for project generator
```

Comment:
Don't use tabs.

Reply:
Nothing has changed, what's different about this line? It's just a space down more.

Reply:
And there still.